src/rundeck/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+name: rundeck
+version: 0.1.0
+home: https://github.com/Deepak-Routray/charts
+source:
+  - https://github.com/Deepak-Routray/rundeck-docker
+appVersion: "1.0"
+description: A Helm chart for Rundeck
+maintainers:
+  - name: DEEPAK ROUTRAY
+
```

Comment:
Please drop this empty line. the maintainer name should be your github username.